src/components/Swap/Output.tsx

```diff
@@ -10,12 +11,12 @@ import { FieldWrapper } from './Input'
 
 export const colorAtom = atom<string | undefined>(undefined)
 
-const OutputWrapper = styled(FieldWrapper)<{ hasColor?: boolean | null }>`
+const OutputWrapper = styled(FieldWrapper)<{ hasColor?: boolean | null; widgetWidthPx: number }>`
```

Comment:
See comment on `InputColumn` - logic should not live within the CSS.
Also note that this may be overriding the CSS for `InputColumn` (which I think is your intention).